docs/platform/howto/byoc/add-customer-info-custom-cloud.rst

```diff
@@ -1,9 +1,13 @@
-Add or remove customer contacts for your custom cloud
-=====================================================
+Add or remove customer contacts for your AWS custom cloud in Aiven
+==================================================================
+
+.. topic:: Custom clouds
+
+    A :doc:`custom cloud </docs/platform/concepts/byoc>` is your own cloud infrastructure integrated with your Aiven organization. Using a custom cloud in Aiven may be the optimal solution if you have specific business needs or project requirements, such as a strict regulatory compliance. Creating and using custom clouds in your Aiven organization requires enabling :doc:`the bring your own cloud (BYOC) feature </docs/platform/concepts/byoc>`. Check the availability of the feature in :ref:`Who is eligible for BYOC <eligible-for-byoc>`. To activate BYOC in your Aiven organization, follow the steps in :doc:`Enable bring your own cloud (BYOC) with Aiven </docs/platform/howto/byoc/enable-byoc>`.
```

Comment:
This could be confusing.
"...is your own cloud infrastructure integrated with your Aiven organization."

Perhaps this is a bit ore clear:
A custom cloud is a secure environment within your cloud provider account to run Aiven managed data services.

If you agree with this change, we should use it wherever we define a custom cloud.